tempto-core/src/main/java/io/trino/tempto/internal/query/CassandraQueryExecutor.java

```diff
@@ -75,7 +79,11 @@
 
     public CassandraQueryExecutor(Configuration configuration)
     {
+        ProgrammaticDriverConfigLoaderBuilder loader = DriverConfigLoader.programmaticBuilder();
+        configuration.getInt("databases.cassandra.request_timeout_seconds")
```

Comment:
nit : Can we make this property in sync with cassandra property like `databases.cassandra.basic.request.timeout_seconds` so it allow us to know the actual property being mapped to.